src/utils/components/ProjectCheckerAlert/ProjectCheckerAlert.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { FC } from 'react';
+
+import { modelToGroupVersionKind, ProjectModel } from '@kubevirt-ui/kubevirt-api/console';
+import Loading from '@kubevirt-utils/components/Loading/Loading';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { K8sResourceKind, ResourceLink } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Alert,
+  AlertVariant,
+  Button,
+  Flex,
+  FlexItem,
+  pluralize,
+  Popover,
+} from '@patternfly/react-core';
+
+type ProjectCheckerAlertProps = {
+  projectsLoaded: boolean;
+  qualifiedProjects: K8sResourceKind[];
+};
+
+const ProjectCheckerAlert: FC<ProjectCheckerAlertProps> = ({
+  projectsLoaded,
+  qualifiedProjects,
```

Comment:
u can add a default value of [] and avoid line 31

Reply:
Fixed!